recipes/urwid/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.3.1" %}
+
+package:
+  name: urwid
+  version: "{{ version }}"
+
+source:
+  fn: urwid-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/u/urwid/urwid-{{ version }}.tar.gz
+  md5: 2e1a005cb31368fe21bfeba2d6ad5a5c
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - python
+  build:
+    - python
+    - setuptools
```

Comment:
Added a PR ( https://github.com/conda-forge/staged-recipes/pull/317 ) to fix that.